utilities/uc-verification-request.vbs

```diff
@@ -58,29 +59,28 @@ changelog_display
 
 'Grabs the case number
 EMConnect ""
+CALL check_for_MAXIS(False)
 CALL MAXIS_case_number_finder (MAXIS_case_number)
 closing_message = "Request for Unemployment Insurance Verification email has been sent." 'setting up closing_message UCriable for possible additions later based on conditions
 '----------------------------------------------------------------------------------------------------Initial dialog
 initial_help_text = "*** Unemployment Compensation ***" & vbNewLine & vbNewLine & "For residents under the age of 18 please see" & vbNewLine & "CM0010.18.01 - MANDATORY VERIFICATIONS - CASH" & vbNewLine & "CM0010.18.02 - MANDATORY VERIFICATIONS - SNAP" & vbNewLine & "Unemployment Insurance benefits are considered countable unearned income for all programs." & vbNewLine & vbNewLine &"NOTE: UC(Unemployment Compensation) and UI(Unemployment Income) are used interchangeably by workers."
 
 '---------------------------------------------------------------------DIALOG
 Dialog1 = "" 'Blanking out previous dialog detail
-BeginDialog Dialog1, 0, 0, 311, 85, "Request for Unemployment Insurance"
-  EditBox 55, 5, 40, 15, MAXIS_case_number
+BeginDialog Dialog1, 0, 0, 296, 100, "Request for Unemployment Insurance"
   ButtonGroup ButtonPressed
-    PushButton 175, 5, 15, 15, "!", initial_help_button
-    PushButton 200, 5, 105, 15, "Unemployment Insurance", HSR_manual_button
-  DropListBox 245, 30, 60, 15, "ES Support"+chr(9)+"1800 Chicago"+chr(9)+"HC in METs", team_email_dropdown
-  CheckBox 10, 35, 25, 10, "CCA", cca_checkbox
-  CheckBox 10, 50, 80, 10, "Other (please specify)", other_checkbox
-  EditBox 95, 45, 45, 15, other_check_editbox
+    OkButton 195, 80, 45, 15
+    CancelButton 245, 80, 45, 15
+  EditBox 85, 5, 50, 15, MAXIS_case_number
```

Comment:
It is a bit odd to have the tab order start with the buttons. It also causes 2 button groups. Recommend having the edit boxes and check boxes before all the buttons. Enter will still press OK if the OK Button is not highlighted.